dist/s_funcs

```diff
@@ -5,7 +5,7 @@ t=__wt.$$
 trap 'rm -f $t' 0 1 2 3 13 15
 
 # List of files to search.
-l=`sed -e '/^[a-z]/!d' -e 's/[	 ].*$//' -e 's,^,../,' filelist`
+l=`sed -e '/^[a-z]/!d' -e 's/[     ].*$//' -e 's,^,../,' filelist`
```

Comment:
We will probably need to revise this regex, maybe we only need one space instead here